src/DictVectors/pdvec.jl

```diff
@@ -567,6 +582,65 @@ function Base.map!(f, dst::PDVec, src::PDVecVals)
     return dst
 end
 
+"""
+    map(f, values(::PDVec))
+
+Out-of-place parallel `map` on values of a [`PDVec`](@ref). Returns a new
+[`PDVec`](@ref). Only defined for `values` as efficiently changing keys in a thread-safe and
+distributed way is not possible.
+"""
+function Base.map(f, src::PDVecVals)
+    return map!(f, copy(src.vector), src)
+end
+
+"""
+    filter!(pred, [dst::PDVec, ], keys(::PDVec))
+    filter!(pred, [dst::PDVec, ], values(::PDVec))
+    filter!(pred, [dst::PDVec, ], pairs(::PDVec))
+
+In-place parallel `filter!` on an iterator over a [`PDVec`](@ref). If `dst` is provided,
+results are written there.
+"""
+function Base.filter!(f::F, src::PDVecIterator) where {F}
```

Comment:
Calling this method directly gives a `MethodError`

Reply:
I would expect the following to work:
```
v = PDVec(BoseFS((1,2,3)) => 4)
filter!(iseven, values(v))
```

Reply:
This should work. Maybe it was bugged earlier, but it's in the tests now.